yarn.lock

```diff
@@ -12556,7 +12556,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"handlebars@npm:^4.7.8":
+"handlebars@npm:^4.7.7, handlebars@npm:^4.7.8":
```

Comment:
Did you manually set this? I'm not familiar with and don't have the guts to edit the lock file.

Reply:
No, I just run `jlpm dlx yarn-berry-deduplicate`. But you could see the changelog the integrity job was asking for by looking at the logs:

![image](https://github.com/user-attachments/assets/02713302-3595-40f8-adc1-6ea8da17b71c)